enos/modules/test_e2e/main.tf

```diff
@@ -122,7 +122,7 @@ resource "enos_local_exec" "run_e2e_test" {
     E2E_AWS_HOST_SET_IPS2         = local.aws_host_set_ips2
   }
 
-  inline = ["PATH=\"${var.local_boundary_dir}:$PATH\" go test -v ${var.test_package} -count=1"]
+  inline = ["PATH=\"${var.local_boundary_dir}:$PATH\" go test -v ${var.test_package} -count=1 -json | tparse -follow -format plain"]
```

Comment:
Weird that the format looks ugly in github actions when it looks fine in CircleCI, maybe worth raising an issue? Also looks like the markdown format should work particularly well for github actions: https://github.com/mfridman/tparse/issues/57?

Reply:
I'll try it and see what it looks like

Reply:
![Screen Shot 2022-12-14 at 8 25 26 PM](https://user-images.githubusercontent.com/2474253/207750631-72ad835e-82c0-4fc7-bdc9-2c870c7d561f.png)
https://github.com/hashicorp/boundary-enterprise/blob/main/.github/workflows/build.yml#L392-L396

Reply:
Created an issue here: https://github.com/mfridman/tparse/issues/85